### PR TITLE
making win_reg module salt-less

### PR DESCRIPTION
**Pull request by MoodyMudit**

### CODE REVIEW

File: tests/unittests/modules/test_reg.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        )
+
+    def test_read_value_unknown_key_error(self):
+        """
+        Tests the read_value function with an unknown key error
+        """
+        mock_error = MagicMock(
+            side_effect=win32api.error(123, "RegOpenKeyEx", "Unknown error")
+        )
+        with patch("salt.utils.win_reg.win32api.RegOpenKeyEx", mock_error):
```
Comment (goravsingal) [inline tests/unittests/modules/test_reg.py:120]: it should be "hubblestack.utils."

File: tests/unittests/modules/test_reg.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+            )
+
+    def test_read_value_unknown_value_error(self):
+        """
+        Tests the read_value function with an unknown value error
+        """
+        mock_error = MagicMock(
+            side_effect=win32api.error(123, "RegQueryValueEx", "Unknown error")
+        )
+        with patch("salt.utils.win_reg.win32api.RegQueryValueEx", mock_error):
```
Comment (goravsingal) [inline tests/unittests/modules/test_reg.py:136]: it should be "hubblestack.utils."

**Review by goravsingal — CHANGES_REQUESTED** [review]

**Review by devagupt — CHANGES_REQUESTED** [review]

File: tests/unittests/modules/test_reg.py (hunk 1 above)
Comment (MoodyMudit) [reply tests/unittests/modules/test_reg.py]: Done.

File: tests/unittests/modules/test_reg.py (hunk 2 above)
Comment (MoodyMudit) [reply tests/unittests/modules/test_reg.py]: Done.

File: tests/unittests/modules/test_reg.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (MoodyMudit) [reply tests/unittests/modules/test_reg.py]: Done.

File: hubblestack/modules/reg.py [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+    ``SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Run``
+
+Value:
+    - There are 3 value names:
+        - `RTHDVCPL`
+        - `NvBackend`
+        - `BTMTrayAgent`
+    - Each value name has a corresponding value
+
+:depends:   - salt.utils.win_reg
```
[the remark this answers is not in the document]
Comment (MoodyMudit) [reply hubblestack/modules/reg.py:66]: Done.

File: tests/unittests/modules/test_reg.py [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+UNICODE_VALUE = (
+    "Unicode Value " "\N{COPYRIGHT SIGN},\N{TRADE MARK SIGN},\N{REGISTERED SIGN}"
+)
+FAKE_KEY = "SOFTWARE\\{}".format(random_string("SaltTesting-", lowercase=False))
+
+
+@skipIf(not HAS_WIN32, "Tests require win32 libraries")
+class WinFunctionsTestCase(TestCase, LoaderModuleMockMixin):
+    """
+    Test cases for salt.modules.reg
```
[the remark this answers is not in the document]
Comment (MoodyMudit) [reply tests/unittests/modules/test_reg.py]: Done.

File: tests/unittests/modules/test_reg.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (MoodyMudit) [reply tests/unittests/modules/test_reg.py]: done.

**Review by devagupt — APPROVED** [review]

**Review by goravsingal — APPROVED** [review]

**Pull request merged**